CMakeLists.txt

```diff
@@ -279,7 +314,7 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
   )
 endif() # TRITON_PYTORCH_DOCKER_BUILD
 
-set(TRITON_PYTORCH_LDFLAGS "")
+set(TRITON_PYTORCH_LDFLAGS "-L.")
```

Comment:
Why is this change needed?

Reply:
Must be a merge error. I will remove that.